### ISSUE TITLE
`jira issue create --no-input` attempts to read stdin and hangs

### ISSUE THREAD

**Issue by realtime-neil** [ankitpokhrel/jira-cli#641]
**Describe the bug**
`jira issue create --no-input` hangs indefinitely, prompting for input.

**Please provide following details**
1. JiraCLI Version:
    ```
    (Version="v1.4.0", GitCommit="", CommitDate="", GoVersion="go1.20.5", Compiler="gc", Platform="linux/amd64")
    ```
2. Are you using Jira cloud or on-premise jira server?
    ```
    SERVER INFO
    -----------
    
    Version:         1001.0.0-SNAPSHOT
    Build Number:    100230
    Deployment Type: Cloud
    Default Locale:  en_US
    ```
3. What operating system are you using? Also mention version.
    ```
    PRETTY_NAME="Ubuntu 22.04.2 LTS"
    NAME="Ubuntu"
    VERSION_ID="22.04"
    VERSION="22.04.2 LTS (Jammy Jellyfish)"
    VERSION_CODENAME=jammy
    ID=ubuntu
    ID_LIKE=debian
    HOME_URL="https://www.ubuntu.com/"
    SUPPORT_URL="https://help.ubuntu.com/"
    BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
    PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
    UBUNTU_CODENAME=jammy
    ```
4. What terminal are you using? Also mention version.
    ```
    $ st -v
    st 0.9
    ```

**To Reproduce**

```
$ nohup jira issue create --no-input >/tmp/nohup.out ; echo $? ; cat /tmp/nohup.out
nohup: ignoring input and redirecting stderr to stdout
1
? Issue type  [Use arrows to move, type to filter]
> Task
  Story
  Bug
  Customer Issue
  Spike
  Initiative
  Customer Request
Error: read /dev/stdin: bad file descriptor
```

**Expected behavior**
* no attempt to read `/dev/stdin`.
* error message about missing information

**Screenshots**
not applicable

**Additional context**
not applicable

**Comment by ankitpokhrel** [comment]
Looks like this happens if we don't pass required params. Thank you for reporting!

```sh
# This one works
$ nohup jira issue create -tTask -s"Issue 641 test" --no-input
```

**Comment by ankitpokhrel** [comment]
This should be fixed with #655
